Studio-Lovelies/GG-JointJustice-Unity
# feature/259 Controls bar shows display value of keys

## Summary

Controls in the controls bar update to display correct key bindings set in the InputActions.

## Before/after screenshots and/or animated gif


## Testing instructions

View the controls bar in the evidence or pause menu. Change a key binding in the InputActions asset. See if they get updated on the controls bar.

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  
closes #259
